### PR TITLE
ci: RTD fix for API listing

### PR DESCRIPTION
**Pull request by paulgessinger**

### CODE REVIEW

**Comment by paulgessinger** [comment]
@benjaminhuth this should bring it back.

File: docs/conf.py [hunk 1]
```diff
@@ -112,7 +115,12 @@
         cwd=cwd,
         env=env,
     )
+    if not api_index_target.exists():
+        shutil.copyfile(cwd / "api/api_index.rst", cwd / "api/api.rst")
```
Comment (benjaminhuth) [inline docs/conf.py:119]:
```suggestion
        shutil.copyfile(cwd / "api/api_index.rst", api_index_target)
```

**Review by benjaminhuth — APPROVED** [review]
Ah so the problem was that on readthedocs the thing isn't run with cmake and thus did not copy the file...
Looks good!

**Comment by paulgessinger** [comment]
I applied your suggestion, can you reapprove?

**Review by benjaminhuth — APPROVED** [review]

**Pull request merged**